contracts/MyDebts.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity ^0.4.0;
+
+contract MyDebts {
+    mapping(address => uint) debts;
+
+    event Lended(address borrower, uint sum);
+    event Repaid(address borrower, uint sum);
+    event Error(string message);
+
+    function lend(uint _sum) public returns(bool) {
+        if (debts[msg.sender] > 0) {
```

Comment:
Why do you need this check? What would happen if you delete it and left just debts[msg.sender] += _sum ?

Reply:
What will happen if someone will try to borrow 2^256 and than borrow 1?